cli/templates/config/README.md

```diff
@@ -0,0 +1,11 @@
+# Code Configuration
+
+Welcome to your very own Grouparoo project! You are awesome for supporting open-source software!
```

Comment:
I'm not sure about "open source" being the main thing we call out here.  Maybe just stop after the welcome?

@andyjih what do you think?